openedx_blockstore_relay/tests/test_transfer_data.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Tests for the `openedx-blockstore-relay` transfer_data module.
+"""
+
+from __future__ import absolute_import, unicode_literals
+
+
+import ddt
+import mock
+from xml.etree import ElementTree
+
+from django.test import TestCase
+from opaque_keys.edx.keys import UsageKey
+from xblock.core import XBlock
+
+from openedx_blockstore_relay.transfer_data import TransferBlock, transfer_to_blockstore
+from openedx_blockstore_relay.test_utils.compat import StubCompat, StaticContent
+
+
+class TestXBlock(XBlock):
```

Comment:
Nit: a docstring would be helpful here. At first glance it's not clear if this is "an XBlock for use in tests" or "a tool to help test an XBlock"

Reply:
Added.